web/src/components/StudentCard.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { Card, Grid, Box } from "@material-ui/core/";
+import { CardActionArea, CardContent, CardMedia } from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import PropTypes from "prop-types";
+import { styled } from "@material-ui/core/styles";
+import { useWideCardMediaStyles } from "@mui-treasury/styles/cardMedia/wide";
+import {
+  Home,
+  Euro,
+  FavoriteBorder,
+  Create,
+  School,
+  Room,
+} from "@material-ui/icons";
+
+const StyledCardActionArea = styled(CardActionArea)({
+  minWidth: "250px",
+  transition: "0.2s",
+  "&:hover": {
+    transform: "scale(1.1)",
+  },
+});
+
+const StyledCard = styled(Card)({
+  backgroundColor: "#FCFCFC",
+  borderRadius: 20,
+  "&:hover": {
+    boxShadow: "0px 10px 10px 0",
+  },
```

Comment:
effect too big imo

Reply:
Changed to "0px 5px 5px 0px". Check again please